tests/connections/GenericHookTest.ts

```diff
@@ -175,7 +178,7 @@ describe("GenericHookConnection", () => {
     });
     it("will handle a hook event with a v2 transformation function", async () => {
         const webhookData = {question: 'What is the meaning of life?', answer: 42};
-        const [connection, mq] = createGenericHook({name: 'test', transformationFunction: V2TFFunction}, {
+        const [connection, mq] = createGenericHook({name: 'test', hookId: 'test', transformationFunction: V2TFFunction}, {
```

Comment:
I believe it's still technically valid to have these without hookIds?

Reply:
This was required due to having made `hookId` mandatory, but that can be reverted.

Reply:
Reverted